azurerm/internal/services/datashare/tests/resource_arm_data_share_dataset_kusto_database_test.go

```diff
@@ -0,0 +1,130 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+)
+
```

Comment:
COuld we add an update test?

Reply:
Sorry, all fields are forcenew. There's no update method.